### ISSUE TITLE
mame setup

### ISSUE THREAD

**Issue by VermillionFury007** [beardypig/guncon3#5]
been trying to use the gcon3 with MAME but without sucess. in the games menu, it gets recognized, but not ingame

**Comment by james-sounds-crazy** [comment]
i've got the guncon3, i've got mame, 
what's stopping me?
i don't know how to install this on windows via the WSL (Windows Subsytem Llinux)  or something

**Comment by beardypig** [comment]
@james-sounds-crazy I do not think this would work with WSL, perhaps if you ran mame inside WSL somehow...
